README.md

```diff
@@ -1,56 +1,30 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# OOP-School-Library
 
-> Description the project.
+> This app comprises a school library app.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
-
-### Run tests
+To get a local copy up and running follow these simple steps.
 
-### Deployment
+To setup the project in your local, in the repo page: click on code (dropdown list) > Download as ZIP. or open terminal of path you want to install project and run this command
+git clone git@github.com/oliverSCZ/OOP-school-library.git
```

Comment:
- [ ] I would recommend you itemize these instructions so that your documentation looks better.

  ![oliver-readme](https://user-images.githubusercontent.com/36542986/157141238-059c4add-68ad-4248-98aa-7b00aa9e7412.jpg)

Reply:
Done